content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -178,36 +231,36 @@ The requirements to run TrueCommand in Docker Desktop for Windows are:
 * CPU support for VM Monitor Mode Extension (VT-c on Intel CPUs).
 * Hyper-V is enabled in Windows 10.
 * 4 GB memory at minimum.
-* [Docker Desktop](https://www.docker.com/products/docker-desktop) needs to be installed in Windows.
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) must be installed in Windows.
```

Comment:
This list of requirements is not following parallel construction so you could reword or make them a separate sentence below. For example, from line177:
The requirements to run TrueCommand in Docker Desktop for Windows are you must have:
* CPU support for VM Monitor Mode Extension (VT-c on Intel CPUS)
* Hyper-V enabled in Windows 10
* 4 GB memory at minimum
* [Docker Desktop](https://www.docker.com/products/docker-desktop installed in Windows 

A list of requirements does not need the period at the end of the bullet point even if it is a complete sentence.